### PR TITLE
OA-32: Spring web app context's config location set to "classpath*:/oauth2login/webApplicationContext.xml".

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket: https://issues.openmrs.org/browse/OA-32

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/oauth2login/web/CustomDispatcherServlet.java [hunk 1]
```diff
@@ -40,8 +40,7 @@ public void setServletContext(ServletContext servletContext) {
 		
 		try {
 			XmlWebApplicationContext appContext = new XmlWebApplicationContext();
-			appContext.setConfigLocation("classpath:webModuleApplicationContext.xml");
-			
+			appContext.setConfigLocation("classpath*:/oauth2login/webApplicationContext.xml");
```
Comment (mks-d) [inline omod/src/main/java/org/openmrs/module/oauth2login/web/CustomDispatcherServlet.java:43]: Don't you want to use [this](https://github.com/openmrs/openmrs-module-oauth2login/blob/37183a76108384eb0484fa509a2ed890619014a3/api/src/main/java/org/openmrs/module/oauth2login/OAuth2LoginConstants.java#L19)?

Comment (Ruhanga) [reply omod/src/main/java/org/openmrs/module/oauth2login/web/CustomDispatcherServlet.java]: Resolved, thanks @mks-d.

**Review by ibacher — APPROVED** [review]
I'm assuming you tested this without the `*` and it was necessary? Otherwise, I think this looks correct.

**Comment by Ruhanga** [comment]
> I'm assuming you tested this without the * and it was necessary? Otherwise, I think this looks correct.
Yes @ibacher. Thanks for the review.

**Pull request merged**